hoangnm/react-native-week-view
# Chore: setup simple tests and github actions

* Simple tests: `App` and `WeekView` render correctly
* Test workflows:
  * `test-main`: test library source code
  * (removed) ~~`test-example`: test example App code by itself~~
  * (removed) ~~`test-example-in-local`: test example App as it would be run by a developer (to avoid issues like this: https://github.com/hoangnm/react-native-week-view/pull/114#discussion_r680345968)~~
* Add simple developing instructions
* Setup yarn scripts for installation in example App (see details in next comment)

ps: I will setup `linter.yml` later, when other PRs are closed (to avoid code conflicts)
